### ISSUE TITLE
terra-date-picker with min/max dates fails WDIO tests because of color contrast issues 

### ISSUE THREAD

**Issue by gustavofulton** [cerner/terra-framework#908]
# Bug Report
`Terra-date-picker` with min/max dates fails WDIO tests because of color contrast issues 

## Description

Using `terra-date-picker` with min/max dates will cause accessibility issues when running WDIO tests because of color contrast issues between the disabled dates(`#cccccc`) and the background(`#ffffff`). 

## Steps to Reproduce

1. Use terra-date-picker component and set either minDate or maxDate.
2. Run wdio test with an action that opens up calendar picker (something like this `browser.click('.DatePicker-module__button__...'`);

## Additional Context / Screenshots

<img width="212" alt="Screen Shot 2019-09-26 at 3 47 25 PM" src="https://user-images.githubusercontent.com/10981250/65776563-06b4a200-e108-11e9-976f-c77b006063c4.png">
<img width="1166" alt="Screen Shot 2019-09-27 at 9 20 47 AM" src="https://user-images.githubusercontent.com/10981250/65776630-264bca80-e108-11e9-8fda-58d4b695d59c.png">

## Expected Behavior

WDIO tests shouldn't fail because of color contrast issues.

## Possible Solution

Posted on the teams channel and it seems _the aria-disabled attribute should be implemented for dithered date values._

## Environment

* Component Name and Version: `terra-date-picker@4.13.0`
* Browser Name and Version: Chrome 
* Node/npm Version: Node 10.16.2, npm 6.9.0
* Operating System and version (desktop or mobile): MacOS

## @ Mentions

@dkasper-was-taken answered to my post on Teams for me to open the bug here.
